osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyCursorParticles.cs

```diff
@@ -84,6 +105,50 @@ protected override void Update()
                 kiaiHitObject = playfield.HitObjectContainer.AliveObjects.FirstOrDefault(isTracking);
 
             kiaiSpewer.Active.Value = kiaiHitObject != null;
+
+            if (relaxing != relaxHitSpewer.Active.Value) relaxHitSpewer.Active.Value = relaxing;
+
+            if (relaxingKeyPressed() && !kiaiSpewer.Active.Value)
+            {
+                relaxHitSpewer.Direction = SpewDirection.None;
+                relaxHitSpewer.Colour = starBreakAdditive;
+                relaxHitSpewer.SpawnKiaiParticles = true;
+            }
+            else
+            {
+                relaxHitSpewer.SpawnKiaiParticles = false;
+            }
```

Comment:
What is even happening here with the `SpawnKiaiParticles` thing. Why does the relax spewer spawn kiai particles _when the kiai spewer doesn't_

Reply:
First, the reason why I don't use `kiaiSpewer` directly is that when `kiaiSpewer.Active` is `false` all the particles that were spawned would be hidden. So I use `relaxHitSpewer` to do his job. But when you are in Kiai mode and `relaxHitSpewer` is active, it would spawn twice more particles. This means I must disable either of them.